src/modules/profile/pages/profileSettings/views/ProfileSettingsNotifications.tsx

```diff
@@ -43,14 +25,10 @@ export const ProfileSettingsNotifications = () => {
             {t('Customize your profile and project notifications.')}
           </Body>
         </VStack>
-        {loading ? (
-          <CreatorNotificationsSkeleton />
-        ) : (
-          <CreatorNotifications creatorNotificationSettings={creatorNotificationSettings} />
-        )}
+        <CreatorNotifications />
         <Divider />
-        {userNotificationSettings && <UserNotifications userNotificationSettings={userNotificationSettings} />}
-        {!loading && <FollowedProjectsList />}
+        <UserNotifications />
+        <FollowedProjectsList />
```

Comment:
Much more cleaner like this too, well done!

Reply:
I did this at first to re-use the hook in the funding flow, and have it update the notification setting real-time, but then I realised it wouldn't work with logged out users, so I had to scrap all that 🥲 but hey, we got a nice refactor out of it :p